docs/extensions/leveraging-ui-components.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Learn how to use the Directus UI components in your extension projects.
+contributors: Esther Agbaje
+---
+
+# Leveraging Directus UI Components
+
+Directus provides a set of reusable UI components that can be used in when building extensions. These components are
+designed to provide a consistent user experience across different projects and extensions.
+
+## Components Playground
+
+Explore the available components on the [Histoire Components Playground](https://components.directus.io/). Here, you can
+interact with different components and experiment with variations before implementing them in your project.
```

Comment:
```suggestion
Explore the available components in our [Directus Components Playground](https://components.directus.io/). Here, you can
see all available components and experiment with variations before implementing them in your extensions.
```

Reply:
I don't think we need to expose the readers to this new term and tool - Histoire.